appmgr/src/bin/embassyd.rs

```diff
@@ -245,22 +249,23 @@ fn main() {
         )
         .get_matches();
 
-    simple_logging::log_to_stderr(match matches.occurrences_of("verbosity") {
-        0 => log::LevelFilter::Off,
-        1 => log::LevelFilter::Error,
-        2 => log::LevelFilter::Warn,
-        3 => log::LevelFilter::Info,
-        4 => log::LevelFilter::Debug,
+    // initializes the bootstrap logger, this will be replaced with the EmbassyLogger later
+    let filter = match matches.occurrences_of("verbosity") {
+        0 => log::LevelFilter::Error,
+        1 => log::LevelFilter::Warn,
+        2 => log::LevelFilter::Info,
+        3 => log::LevelFilter::Debug,
         _ => log::LevelFilter::Trace,
-    });
+    };
+    simple_logging::log_to_stderr(filter);
```

Comment:
This is necessary in case there are error logs that happen in the methods that are used to bootstrap the logger, at least they won't be blackholed with this. It will be replaced during the `new` function for the `EmbassyLogger`